pkg/server/admin_test.go

```diff
@@ -346,6 +349,111 @@ func TestAdminAPIDatabaseSQLInjection(t *testing.T) {
 	}
 }
 
+func TestAdminAPINonTableStats(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	testCluster := serverutils.StartTestCluster(t, 3, base.TestClusterArgs{})
+	defer testCluster.Stopper().Stop(context.Background())
+	s := testCluster.Server(0)
+
+	// Skip TableStatsResponse.Stats comparison, since it includes data which
+	// aren't consistent (time, bytes).
+	expectedResponse := serverpb.NonTableStatsResponse{
+		TimeSeriesStats: &serverpb.TableStatsResponse{
+			RangeCount:   1,
+			ReplicaCount: 3,
+			NodeCount:    3,
+		},
+		InternalUseStats: &serverpb.TableStatsResponse{
+			RangeCount:   9,
+			ReplicaCount: 15,
+			NodeCount:    3,
+		},
+	}
+
+	var resp serverpb.NonTableStatsResponse
+	if err := getAdminJSONProto(s, "nontablestats", &resp); err != nil {
+		t.Fatal(err)
+	}
+
+	assertExpectedStatsResponse := func(expected, actual *serverpb.TableStatsResponse) {
+		assert.Equal(t, expected.RangeCount, actual.RangeCount)
+		assert.Equal(t, expected.ReplicaCount, actual.ReplicaCount)
+		assert.Equal(t, expected.NodeCount, actual.NodeCount)
+	}
+
+	assertExpectedStatsResponse(expectedResponse.TimeSeriesStats, resp.TimeSeriesStats)
+	assertExpectedStatsResponse(expectedResponse.InternalUseStats, resp.InternalUseStats)
+}
+
+// TODO(celia) - I expect all the ranges listed on the Database page to equal
+// the RangeCount returned from doing a span on [LocalMax, MaxKey). For a cluster
+// with no user data, all the ranges on the Databases page consist of:
+// 1) the total ranges listed for the system database
+// 2) the total ranges listed for the Non-Table data
+func TestRangeCount_MissingOneRange(t *testing.T) {
```

Comment:
The ranges on the Databases page should now equal the total number of ranges on the Overview page. However, I'm still missing one.. ?

![image](https://user-images.githubusercontent.com/3051672/47819256-4cb68980-dd30-11e8-877a-14b6a51dc3c1.png)


![image](https://user-images.githubusercontent.com/3051672/47819223-3e686d80-dd30-11e8-9c3e-6a6d6428871d.png)